src/main/java/org/skriptlang/skript/bukkit/damagesource/elements/CondWasIndirect.java

```diff
@@ -0,0 +1,72 @@
+package org.skriptlang.skript.bukkit.damagesource.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.util.Kleenean;
+import org.bukkit.damage.DamageSource;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.damagesource.DamageSourceExperiment;
+
+@Name("Damage Source - Was Indirectly Caused")
+@Description({
+	"Whether the damage from a damage source was indirectly caused.",
+	"Vanilla damage sources are considered indirect if the 'causing entity' and the 'direct entity' are not the same. "
+		+ "(i.e. taking damage from an arrow that was shot by an entity)"
+})
+@Example("""
+	on damage:
+		if event-damage source was indirectly caused:
+	""")
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.20.4+")
+public class CondWasIndirect extends PropertyCondition<DamageSource> implements DamageSourceExperiment {
+
+	static {
+		Skript.registerCondition(CondWasIndirect.class,
+			"%damagesources% (was|were) ([:in]directly caused|caused [:in]directly)",
+			"%damagesources% (was not| wasn't|were not|weren't) ([:in]directly caused|caused [:in]directly)"
```

Comment:
i'm not 100% happy with this but i think it's good enough
my discontent comes from it implying the source was caused indirectly, rather than the damage being caused indirectly